ansible/library/lldp_facts.py

```diff
@@ -90,7 +90,8 @@ def __init__(self,dotprefix=False):
             dp = ""
 
         # From IF-MIB
-        self.if_descr       = dp + ".0.8802.1.1.2.1.3.7.1.3"
+        #ifdescr is common support, replace the lldpportid
+        self.if_descr       = dp + ".3.6.1.2.1.2.2.1.2"
```

Comment:
Will this change affect the existing LLDP testing?

Reply:
There is not any side effective with existing LLDP case. The main purpose is get interfaces collection from dut, and use these data for sanity check. I have double confirmed this based on running existing testcases.